website/docs/docs/cloud/git/connect-github.md

```diff
@@ -48,15 +47,15 @@ To connect your dbt Cloud account to your GitHub account:
    - Read and write access to Workflows
 
 6. Once you grant access to the app, you will be redirected back to dbt Cloud and shown a linked account success state. You are now personally authenticated. 
-7. Ask your team members to [personally authenticate](/docs/cloud/git/connect-github#personally-authenticate-with-github) by connecting their GitHub profiles.
+7. Ask your team members to individually authenticate by connecting their [personal GitHub profiles](#authenticate-your-personal-github-account).
 
 ## Limiting repository access in GitHub
 If you are your GitHub organization owner, you can also configure the dbt Cloud GitHub application to have access to only select repositories. This configuration must be done in GitHub, but we provide an easy link in dbt Cloud to start this process.
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-github/configure-github.png" title="Configuring the dbt Cloud app"/>
 
-## Personally authenticate with GitHub
+## Authenticate your personal GitHub account
```

Comment:
have links been checked since it's a heading change? iirc we need to do this for anchor links (ie links to sections on page) bc our link checker doesn't catch them

Reply:
Yes! There were two links that used the old header and I changed them. One on this page and one on another.